File: src/LCMandHCF.py
t = int(input())
for _ in range(t):
    a, b = map(int,input().split())
    x, y = max(a, b), min(a, b)
    while y > 0:
        x, y = y, x % y
    hcf = x

    # Calculate LCM
    lcm = (a * b) // hcf
    print(lcm, hcf, sep=" ")

File: src/LCMandHCF.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        long t = sc.nextLong();
        while(t-- > 0){
            long a = sc.nextLong(), b = sc.nextLong();
            long x, y;
            if(a > b){
                x = a;
            }
            else{
                x = b;
            }

            if(a<b){
                y = a;
            }
            else{
                y = b;
            } 
            
            while(y > 0){
                long temp = y;
                y = x % y;
                x = temp;
            }
            long hcf = x;
            long lcm = (a*b)/hcf;
            System.out.println(lcm + " " + hcf);
        }
    }
}